benchmarks/engines.toml

```
@@ -426,6 +426,32 @@
     bin = "cargo"
     args = ["clean"]
 
+# D's regex library, bundled with its standard library.
+#
+# URL: https://dlang.org/phobos/std_regex.html
+[[engine]]
+  name = "d/dmd/std-regex"
+  cwd = "../engines/d"
+  [engine.version]
+    bin = "./main"
+    args = ["--version"]
+  [engine.run]
+    bin = "./main"
+  [[engine.dependency]]
+    bin = "dub"
+    args = ["--version"]
+    regex = '(?m)^DUB'
+  [[engine.dependency]]
+    bin = "dmd"
+    args = ["--version"]
+    regex = '(?m)^DMD'
+  [[engine.build]]
+    bin = "dub"
+    args = ["build", "-b", "release"]
```

```suggestion
  [[engine.build]]
    bin = "dub"
    args = ["build", "-b", "release", "--compiler=dmd"]
```
It's probably best to tell dub specifically to use the dmd compiler when building here, since the engine is named after dmd. Else it might pickup ldc by mistake.

---

Excellent catch. Thank you.